assignments/subscriptions/b72ab7b7-723f-4b18-b6f6-03b0f2c6a1bb/assign.export_defender_logs_eventhub.json

```diff
@@ -0,0 +1,31 @@
+{
+  "properties": {
+    "metadata": {
+      "assignedBy": "PlatOps"
+    },
+    "description": "Deploy export to Event Hub for Microsoft Defender for Cloud data",
+    "displayName": "HMCTS  - Deploy export to Event Hub for Microsoft Defender for Cloud - CFT SBOX (DCD-CFTAPPS-SBOX)",
+    "enforcementMode": "Default",
+    "nonComplianceMessages": [],
+    "notScopes": [],
+    "parameters": {
+      "resourceGroupName":{
+          "value": "rg-export-defender-eventhub-uksouth"
+      },
```

Comment:
Do we need to apply the "resourceGroupLocation" to this for the default to be applied?